src/test/kotlin/nl/info/zac/authentication/UserPrincipalFilterTest.kt

```diff
@@ -193,6 +201,9 @@ class UserPrincipalFilterTest : BehaviorSpec({
         every { accessToken.getStringListClaimValue("group_membership") } returns groups
         every { zaakafhandelParameterService.listZaakafhandelParameters() } returns zaakafhandelParameters
         every { httpSession.setAttribute(any(), any()) } just runs
+        every {
+            pabcClientService.getApplicationRoles(any())
+        } returns mockk(relaxed = true)
```

Comment:
why return a `mockk(relaxed = true)` here and not a real mock object that you explicitly create (using a fixture)?